### PR TITLE
Tests: Migrate stored, amp, cache tests to generic

### PR DESCRIPTION
**Pull request by ValeriiBarsuk**
Tests: Migrate stored, amp, cache tests to generic

### CODE REVIEW

File: src/test/resources/org/prebid/server/it/storedrequests/test-amp-stored-request.json [hunk 1]
```diff
@@ -6,20 +6,27 @@
...
+          "gam-key2": "val2"
+        },
+        "tid": "6b8762d8-26ec-45bc-821b-8d6a72fdcd55",
+        "generic": {
           "accountId": 5001,
           "siteId": 6001,
           "zoneId": 7001
         },
-        "appnexus": {
+        "genericAlias": {
```
Comment (SerhiiNahornyi) [inline src/test/resources/org/prebid/server/it/storedrequests/test-amp-stored-request.json:29]: We don't need appnexus properties 
Just some `"property1":"value1"`

File: src/test/resources/org/prebid/server/it/storedimps/test-genericAlias-stored-video-cache-update.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     ]
   },
   "ext": {
-    "rubicon": {
+    "genericAlias": {
```
Comment (SerhiiNahornyi) [inline src/test/resources/org/prebid/server/it/storedimps/test-genericAlias-stored-video-cache-update.json:21]: We don't need rubicon properties
just Just some `"property2":"value2"`

**Review by SerhiiNahornyi — CHANGES_REQUESTED** [review]

File: src/test/resources/org/prebid/server/it/storedrequests/test-amp-stored-request.json (hunk 1 above)
Comment (ValeriiBarsuk) [reply src/test/resources/org/prebid/server/it/storedrequests/test-amp-stored-request.json:29]: Done

File: src/test/resources/org/prebid/server/it/storedimps/test-genericAlias-stored-video-cache-update.json (hunk 2 above)
Comment (ValeriiBarsuk) [reply src/test/resources/org/prebid/server/it/storedimps/test-genericAlias-stored-video-cache-update.json:21]: Done

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**